### PR TITLE
Fix: Update addon name in reame.txt

### PR DESCRIPTION
**Pull request by kjohnson**
## Description

This PR includes the file `readme.txt` in the list of files parsed for replacements and adds the `ADDON_NAME` replacement tag to the header text.

Also, this removes `wordimpress` from the default contributors list.

## Testing Instructions

Building a new add-on should update the `readme.txt` to include the `ADDON_NAME`.

## Pre-review Checklist

-   [x] Acceptance criteria satisfied and marked in related issue
-   [ ] Relevant `@unreleased` tags included in DocBlocks
-   [ ] Includes unit and/or end-to-end tests
-   [x] [Self Review](https://give.gitbook.io/development-manual/devops/github/code-reviews#self-review) of code and UX completed

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
-=== Give - Addon Boilerplate ===
-Contributors: givewp, wordimpress, dlocc, webdevmattcrom
+=== Give - ADDON_NAME ===
+Contributors: givewp, dlocc, webdevmattcrom
 Donate link: https://givewp.com/
 Tags: givewp, donation, donations, donation plugin, wordpress donation plugin, wp donation, donors, display donors, give donors, anonymous donations
 Requires at least: 4.8
```
Comment (ravinderk) [inline readme.txt]: @kjohnson can we update this?

Comment (kjohnson) [reply readme.txt]: @ravinderk updated to match Give core.

**Review by ravinderk — APPROVED** [review]

File: readme.txt [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
+Contributors: givewp, dlocc, webdevmattcrom
 Donate link: https://givewp.com/
 Tags: givewp, donation, donations, donation plugin, wordpress donation plugin, wp donation, donors, display donors, give donors, anonymous donations
-Requires at least: 4.8
-Tested up to: 5.5
+Requires at least: 4.9
+Tested up to: 5.8
 Requires PHP: 5.6
 Stable tag: 1.0
 Requires Give: 2.8.0
```
Comment (ravinderk) [inline readme.txt]: @kjohnson one more minor request, can we update this to the latest?

Comment (kjohnson) [reply readme.txt]: Updated to the current version of GiveWP.

**Pull request merged**